### PR TITLE
#2 Home page

### PR DESCRIPTION
**Pull request by 375291601334**
What was done?
Add content (header and article) to the home tab. The autor of the day will render, when card of director will be done.
Change site header to Театральные режиссеры Беларуси.
What problems did you have?
There is no director card yet

What wasn-t done and why?
The author of the day - because there is no director card yet

### CODE REVIEW

**Comment by 375291601334** [comment]
It's wrong PR

File: theatre-directors/src/pages/Home.jsx [hunk 1]
```diff
@@ -1,9 +1,40 @@
 import React from 'react';
+import directorCard from '../components/DirectorsList_Search/SearchResultCard';
```
Comment (AlenaKharytonchyk) [inline theatre-directors/src/pages/Home.jsx]: DirectorCard  - компонент пишем с большой буквы

File: theatre-directors/src/pages/Home.jsx [hunk 2]
```diff
@@ -1,9 +1,40 @@
...
+          театры республики, практически во всех костелах на Рождество также можно увидеть
+          представление в жанре батлейки.
+        </p>
+        <p>
+          Сейчас Беларусь — страна с давней и хорошо развитой театральной культурой. Свой театр
+          есть в каждом областном центре. А наш сайт призван познакомить Вас с наиболее известными
+          теаральными режиссерами нашей страны. Вот один из них:
+        </p>
+      </article>
+      {directorCard()}
```
Comment (AlenaKharytonchyk) [inline theatre-directors/src/pages/Home.jsx]: следует вместо  {directorCard()} использовать тег <DirectorCard />

**Review by AlenaKharytonchyk — CHANGES_REQUESTED** [review]
Настя, маленькие изменения)поправь, пожалуйста

File: theatre-directors/src/pages/Home.jsx (hunk 2 above)
Comment (375291601334) [reply theatre-directors/src/pages/Home.jsx]: исправила в fix:name of component

File: theatre-directors/src/pages/Home.jsx (hunk 1 above)
Comment (375291601334) [reply theatre-directors/src/pages/Home.jsx]: исправила в fix:name of component

**Pull request merged**